README.md

```diff
@@ -328,7 +231,6 @@ Java is a registered trademark of Oracle and/or its affiliates.
 [maven-hbase-client-maven-search]: http://search.maven.org/#search%7Cga%7C1%7Cg:com.google.cloud.bigtable
 [stackoverflow-shield]: https://img.shields.io/badge/stackoverflow-google--cloud--bigtable-blue.svg
 [stackoverflow-link]: http://stackoverflow.com/search?q=[google-cloud-bigtable]
-[integrations]: https://cloud.google.com/bigtable/docs/integrations
 [maven-examples-repo]: https://github.com/GoogleCloudPlatform/cloud-bigtable-examples
```

Comment:
this (and others) seem to be unused and can be removed

Reply:
done